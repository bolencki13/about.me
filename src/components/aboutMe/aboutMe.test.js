import React from 'react'
import renderer from 'react-test-renderer'
import AboutMe from './'

it('renders without crashing', () => {
  const component = renderer.create(
    <AboutMe/>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
