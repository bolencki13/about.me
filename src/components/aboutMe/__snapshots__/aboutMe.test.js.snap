// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<section
  className="container"
  id="bto-about-me"
>
  <div
    className="row h-100"
  >
    <div
      className="col-12 text-center align-self-center"
    >
      <img
        alt="of me"
        className="rounded"
        src="https://media.licdn.com/dms/image/C4E03AQFG0iHhtDVSvg/profile-displayphoto-shrink_200_200/0?e=1531353600&v=beta&t=6PWD8arccHJ1qvMoEWNHe1u6_RH21kN2wRTvd-D1Jmo"
      />
      <h1>
        Hi, I'm Brian. I do a number of things.
      </h1>
      <ul
        className="list-inline"
      >
        <li
          className="list-inline-item"
        >
          Eagle Scout -
        </li>
        <li
          className="list-inline-item"
        >
          Developer -
        </li>
        <li
          className="list-inline-item"
        >
          Entrepreneur
        </li>
      </ul>
      <a
        className="btn btn-dark"
        href="#projects"
      >
        My Projects
      </a>
    </div>
  </div>
</section>
`;
